commands/commands.go

```diff
@@ -339,7 +339,31 @@ func quote(ctx context.Context, s *discordgo.Session, m *discordgo.MessageCreate
 			log.WithFields(ctx.Value(logKey).(log.Fields)).Error("Error getting messages")
 			return
 		}
-		message := messages.Get(rand.Intn(messages.Len()))
+
+		// quote takes into consideration nubmer of reacts for considering which message to quote, more reactions means higher chance
+		// messageDefaultWeight is relative to weight of a single reaction
+		const messageDefaultWeight uint = 1
```

Comment:
Maybe make the const defined at the top of the file. Or even better make it a viper config in `config/defaults.go`